xml/decoders/Digitrax_0SFX_Series7.xsl

```diff
@@ -272,71 +272,37 @@
         <label>   Effect:</label>
     </variable>
 
-    <variable item="XF{$index} Qual" CV="{$CVindex}.235" mask="XXXXVVVV" default="0">
+    <variable item="XF{$index} Qual" CV="{$CVindex}.235" mask="VVVVXXXX" default="0">
         <enumVal xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder.xsd">
-          <enumChoice choice="Forward direction, ON with function ON, phase A">
-            <choice>Forward direction, ON with function ON, phase A</choice>
-            <choice xml:lang="it">Direzione Avanti, ON con Funz.ON, fase A</choice>
+          <enumChoice value="0">
+            <choice>On/OFF lead, controlled by Function output line maps</choice>
           </enumChoice>
-          <enumChoice choice="Reverse direction, ON with function ON, phase B">
-            <choice>Reverse direction, ON with function ON, phase B</choice>
-            <choice xml:lang="it">Direzione Indietro, ON con Funz.ON, fase B</choice>
+          <enumChoice value="1">
+            <choice>On/OFF lead, controlled by Function output line maps</choice>
           </enumChoice>
```

Comment:
Maybe I don't understand what's happening here, but it seems that you have two choices with the same text. The old code referred to "A" and "B" but the new code doesn't seem to do that.

The same applies to enumChoice value 4 and 5.

Reply:
Agree that's really, really odd.  But it's what the Digitrax documentation says.  Apparently one of the encoded bits doesn't actually do anything.